examples/now/pages/index.vue

```diff
@@ -66,9 +66,14 @@
               </b-upload>
               <span
                 v-if="file"
```

Comment:
I think we should hide "Click to upload" when user provide file (if it support only one file attachment)

```vue
...
<b-upload
  v-if="!file"
  :name="field.name"
  v-model="file"
  :required="field.required"
>
  <a class="tag is-secondary">
    <b-icon icon="upload" />
    <span>
      Click to upload
    </span>
  </a>
</b-upload>
<span
  v-else
  class="tag is-primary"
>
  {{ file.name }}
  <button
    @click="deleteFile()"
    class="delete is-small"
    type="button"
  />
</span>
...
```

Reply:
I used `v-show="!file"` instead since `v-if="!file"` prevents uploaded file from being sent.

Reply:
After that you can add `v-else` instead of `v-if="file"`

Reply:
Yeah, You're right. You need to use v-show to keep this `input` value in form.

Keep this in mind. In Javascript we have:
```javascript
const test = null
test === true // return false
test === false // return false
!test === true // return true
!test === false // return false
```
So, `v-if="file"` will return true only if file is filled. If it's `null` it wont render anything